cpp/include/IceUtil/Config.h

```diff
@@ -237,7 +237,7 @@ typedef long Int64;
 #    define ICE_INT64(n) n##L
 #    define ICE_INT64_FORMAT "%ld"
 #else
-typedef long long Int64;
+// typedef int64_t64;
```

Comment:
This works only because you didn't build on a 32-bit Linux system.

Reply:
I've added this back for now, will remove in a follow up cleanup PR